docs/package.json

```diff
@@ -3,20 +3,26 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
+    "dev": "yarn bundle-next-cloudinary && next dev",
+    "build": "yarn bundle-next-cloudinary && next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "bundle-next-cloudinary": "tsup"
   },
   "dependencies": {
-    "@vercel/analytics": "^0.1.5",
-    "cloudinary": "^1.32.0",
-    "next": "^13.0.6",
-    "nextra": "^2.0.0",
-    "nextra-theme-docs": "^2.0.0",
+    "@codesandbox/sandpack-react": "^2.0.21",
```

Comment:
I would recommend upgrading Sandpack to the latest version. It will look faster for Next.js templates